### DISCUSSION TITLE
Default settings

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#308] [General]
### What settings do you think should be changed by default and why?
Write the name of the setting or patch, and let others put a reaction: **LIKE** if you agree and **DISLIKE** if you disagree.

> [!NOTE]
> You can also suggest moving settings from one section to another or anything related to settings.

**Motivation:** _Save people time by enabling/disabling settings that most of us want by default._ AND _Enabling some options, such as hiding or adding various elements, may slow down the application._

**Comment by anddea** [comment]
Bring back all buttons under player, under channel name

Current default: segmented `like` and `dislike`, `save to playlist` buttons
Should be: all available buttons

**Reply by anddea** [reply]
**Сounter idea:** Hide only rewards, shop, thanks buttons

**Comment by anddea** [comment]
Bring back all action bar/quick action buttons under seekbar in fullscreen

Current default: none
Should be: all of them

**Reply by Wojtaz0w** [reply]
YES! I ALMOST UNINSTALLED REVANCED BECAUSE OF THIS

**Reply by anddea** [reply]
Please put your reaction so it's easier to track. 

Although, I disabled it in the first place because it took up a lot of space and it was impossible to concentrate on the content, especially when you double-tap to seek, it shows seekbar in the middle of the screen.

**Comment by anddea** [comment]
Visual preferences icon: Use gear icon for RVX settings instead of puzzle/extension icon.

**Reply by Francesco146** [reply]
sorry to bother you, I'm getting this warning (not recently, it's just that I just now realized it):

```bash
WARNING: Disable auto HDR brightness is not supported in this version. Use YouTube 19.08.36 or earlier (Failed to resolve HDRBrightnessFingerprint)
```

which patch does this? if this happens, does the entire patch fail? If so, is it possible to completely remove the patch without losing functionality? ty!

**Reply by anddea** [reply]
This patch is used only for old versions, and if it's enabled for new ones it will raise this warning. There's nothing critical because hdr auto brightness is obsolete now in new versions.

**Reply by Francesco146** [reply]
> This patch is used only for old versions

which patch? is the one called `Disable HDR video`? because I don't see a patch named `Disable auto HDR brightness`.

thank you for the infos tho!

**Reply by anddea** [reply]
No, it's a different patch, it's just a part of Swipe controls patch.

https://github.com/anddea/revanced-patches/blob/9f783ba5b018476524c1f2abee4f80d5054c73fa/src/main/kotlin/app/revanced/patches/youtube/swipe/controls/SwipeControlsPatch.kt#L134-L159

**Reply by Francesco146** [reply]
understood, thanks!

**Reply by Francesco146** [reply]
> Visual preferences icon: Use gear icon for RVX settings instead of puzzle/extension icon.

migth as well create an outline icon out of the ReVanced logo. it migth be more appropriate
